<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Question #42:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="questionTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="internalAnserLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>134</y>
      <width>57</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="answerTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>251</width>
      <height>82</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="voiceRecButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>134</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextQuestionButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkAnswerButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Answer</string>
    </property>
   </widget>
   <widget class="QLabel" name="internalRightAnswerlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Answer:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rightAnswerTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>251</width>
      <height>82</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerStatusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your answer is wrong</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
